public/components/metrics/view/metrics_grid.tsx

```diff
@@ -3,142 +3,124 @@
 
 import './metrics_grid.scss';
+import { coreRefs } from '../../../framework/core_refs';
+import { PROMQL_METRIC_SUBTYPE } from '../../../../common/constants/shared';
+import { MetricsEditInline } from '../sidebar/metrics_edit_inline';
+import { EmptyMetricsView } from './empty_view';
 
 // HOC container to provide dynamic width for Grid layout
-const ResponsiveGridLayout = WidthProvider(Responsive);
 
 interface MetricsGridProps {
   chrome: CoreStart['chrome'];
-  panelVisualizations: MetricType[];
-  setPanelVisualizations: React.Dispatch<React.SetStateAction<MetricType[]>>;
-  editMode: boolean;
   moveToEvents: (savedVisualizationId: string) => any;
-  editActionType: string;
-  setEditActionType: React.Dispatch<React.SetStateAction<string>>;
 }
 
-export const MetricsGrid = ({
-  chrome,
-  panelVisualizations,
-  setPanelVisualizations,
-  editMode,
-  moveToEvents,
-  editActionType,
-  setEditActionType,
-}: MetricsGridProps) => {
-  // Redux tools
-  const dispatch = useDispatch();
-  const dateSpanFilter = useSelector(dateSpanFilterSelector);
-  const refresh = useSelector(refreshSelector);
-
-  const updateLayout = (metric: any) => dispatch(updateMetricsLayout(metric));
-  const handleRemoveMetric = (metric: any) => {
-    dispatch(deSelectMetric(metric));
-  };
+const visualizationFromMetric = (metric, dateSpanFilter): SavedVisualizationType => ({
+  ...metric,
+  query: updateCatalogVisualizationQuery({ ...metric, ...dateSpanFilter }),
+  queryMetaData: {
+    catalogSourceName: metric.catalogSourceName,
+    catalogTableName: metric.catalogTableName,
+    aggregation: metric.aggregation,
+    attributesGroupBy: metric.attributesGroupBy,
+  },
+
+  timeField: '@timestamp',
+  selected_date_range: dateSpanFilter,
+  userConfigs: {
+    dataConfig: {
+      type: 'line',
+      fillOpacity: 0,
+      lineWidth: 2,
+    },
+    layout: { height: 280 },
+  },
+});
+
+const navigateToEventExplorerVisualization = (savedVisualizationId: string) => {
+  window.location.assign(`${observabilityLogsID}#/explorer/${savedVisualizationId}`);
+};
 
-  const [currentLayout, setCurrentLayout] = useState<Layout[]>([]);
-  const [postEditLayout, setPostEditLayout] = useState<Layout[]>([]);
-  const [gridData, setGridData] = useState(panelVisualizations.map(() => <></>));
-  const [removeMetricsList, setRemoveMetricsList] = useState<Array<{ id: string }>>([]);
-  const isLocked = useObservable(chrome.getIsNavDrawerLocked$());
+export const InnerGridVisualization = ({ id, idx, dateSpanFilter, metric, refresh }) => {
+  if (!metric) return <></>;
 
-  // Reset Size of Visualizations when layout is changed
-  const layoutChanged = (currLayouts: Layout[], allLayouts: Layouts) => {
-    window.dispatchEvent(new Event('resize'));
-    setPostEditLayout(currLayouts);
-  };
-
-  const loadVizComponents = () => {
-    const gridDataComps = panelVisualizations.map((panelVisualization: MetricType, index) => (
+  console.log('vis', { metric });
```

Comment:
Seems like a leftover debug log, there's also a commented line of code a little below on 78.